Cloud Application Manager/Datacenter Edition/camdce-migration.md

```diff
@@ -1,12 +1,15 @@
 {{{
 "title": "Deploy the New Appliance and Import the Data from the Prior Version",
-"date": "09-01-2016",
+"date": "05-24-2019",
 "author": "",
 "attachments": [],
 "contentIsHTML": false
 }}}
 
-### Deploy the New Appliance and Import the Data from the Prior Version
+### Deploy the New Datacenter Edition (appliance) and Import the Data from the Prior Version
```

Comment:
Please, add ToC, Overview, Audience and Prereqs as initial sections in all articles